ckanext/datapreview/transform/base.py

```diff
@@ -12,9 +12,9 @@ def find_transformer(extension=None, mime_type=None):
 
     info = None
     for trans in transformers:
-        if extension and extension in trans["extensions"]:
+        if extension and extension.lower() in trans["extensions"]:
             info = trans
-        if mime_type and mime_type in trans["mime_types"]:
+        if mime_type and mime_type.lower().startswith(trans["mime_types"]):
```

Comment:
You're right, it cannot take a list of strings. It's a tuple of strings. https://docs.python.org/2/library/stdtypes.html#str.startswith